.gitlab-ci.yml

```diff
@@ -188,6 +222,7 @@ publish-docs:
 
 build-android:
   stage:                           optional
+  only:                            *releaseable_branches
```

Comment:
If you do this, then untested android builds will block releases.

Reply:
MB switch stage to test?

Reply:
ok, will place it to `test` phase